code/modules/events/_event.dm

```diff
@@ -104,8 +109,8 @@
 	triggering = TRUE
 
 	if (alert_observers)
-		message_admins("Random Event triggering in [RANDOM_EVENT_ADMIN_INTERVENTION_TIME] seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
-		sleep(RANDOM_EVENT_ADMIN_INTERVENTION_TIME SECONDS)
+		message_admins("Random Event triggering in 10 seconds: [name] (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
+		sleep(100)
```

Comment:
No no, keep `RANDOM_EVENT_ADMIN_INTERVENTION_TIME` but have that be in seconds.

Reply:
done like you said